esrally/driver/driver.py

```diff
@@ -610,6 +610,7 @@ def prepare_telemetry(self, es, enable):
                 telemetry.TransformStats(telemetry_params, es, self.metrics_store),
                 telemetry.SearchableSnapshotsStats(telemetry_params, es, self.metrics_store),
                 telemetry.DataStreamStats(telemetry_params, es, self.metrics_store),
+                telemetry.MasterNodeStats(telemetry_params, es, self.metrics_store),
```

Comment:
As above, if you want this to be an internal telemetry device that runs by default then you can just pass `es_default` in as below:
https://github.com/elastic/rally/blob/4355771a7d9fa027b83331afc2d5e669942b9a0e/esrally/driver/driver.py#L601-L603

Reply:
I did that, thanks! But note that `es_default` vs. `es` is not about about internal/always-on vs. optional: https://elastic.slack.com/archives/C0JGTNK0S/p1631778220222700